website-1.x/blog/2021-03-09-releasing-docusaurus-i18n.md

```diff
@@ -0,0 +1,94 @@
+
+Docusaurus 2 has now reached **full feature parity with Docusaurus 1**, and a first **beta** is on its way.
+
+**We went the extra mile**, and the new i18n support is **even better than in Docusaurus 1**.
+
+In this post, we will present you the **translation workflow**, explain some **design decisions** and **showcase early adopter sites** that just went live: [Jest](https://jestjs.io/), [Redwood](https://learn.redwoodjs.com/), and [Datagit](https://datagit.ir/).
+
+<!--truncate-->
+
+## Translate your site
+
+You can translate a Docusaurus site in **3 simple steps**:
+
+1. **Configure**: declare the default locale and alternative locales in `docusaurus.config.js`.
+2. **Translate**: put the translation files at the correct file-system location.
+3. **Deploy**: build and deploy your site using a single or multi-domain strategy.
+
+The i18n support is **very flexible** and based on the **file-system**.
+
+The **[i18n tutorial](https://v2.docusaurus.io/docs/i18n/tutorial)** is the best way to get started, and we provide help to use **[Git](https://v2.docusaurus.io/docs/i18n/git)** or **[Crowdin](https://v2.docusaurus.io/docs/i18n/crowdin)**.
+
+## Design decisions
+
+The goals of the Docusaurus i18n system are:
+
+- **Simple**: just put the translated files in the correct file-system location.
+- **Flexible translation workflows**: based on Git (monorepo, forks, or submodules), SaaS software, FTP...
+- **Flexible deployment options**: single, multiple domains, or hybrid.
+- **Modular**: allow plugin authors to provide i18n support.
+- **Low-overhead runtime**: documentation is mostly static and does not require a heavy JS library or polyfills.
+- **Scalable build-times**: allow building and deploying localized sites independently.
+- **Localize assets**: an image of your site might contain text that should be translated.
+- **No coupling**: not forced to use any SaaS, yet the integration is possible.
+- **Easy to use with [Crowdin](https://crowdin.com/)**: multiple Docusaurus 1 sites use Crowdin, and should be able to migrate to v2.
+- **Good SEO defaults**: setting useful SEO headers like [`hreflang`](https://developers.google.com/search/docs/advanced/crawling/localized-versions) for you.
+- **RTL support**: locales reading right-to-left (Arabic, Hebrew...) should be easy to use.
+- **Default translations**: theme labels are translated for you in [many languages](https://github.com/facebook/docusaurus/tree/master/packages/docusaurus-theme-classic/codeTranslations).
+
+## Showcase
+
+We worked with a few volunteers to **dogfood the i18n support** before releasing it.
+
+### Jest: upgrading Docusaurus
+
+A **translated Docusaurus 1 site** should be **able to upgrade** to Docusaurus 2.
+
+**[Jest](https://jestjs.io)** was one of those sites. The Docusaurus 2 migration was successful and the new site is now deployed in production ([announcement post](https://jestjs.io/blog/2021/03/09/jest-website-upgrade)).
```

Comment:
"Jest, using Crowdin, was one of those sites"

Reply:
fixed